docs/src/test-parallel-js.md

```diff
@@ -125,6 +125,27 @@ test('runs second', async () => {
 });
 ```
 
+## Override execution mode
+
+If you opt-in all tests into fully-parallel mode in the configuration file:
+
+```js title="playwright.config.ts"
+import { defineConfig } from '@playwright/test';
+
+export default defineConfig({
+  fullyParallel: true,
+});
+```
+
+Sometimes you'll want to run tests in order, the default setting. You can override the mode per describe :
+```js
+test.describe('runs in parallel with other describes', () => {
+  test.describe.configure({ mode: 'default' });
+  test('in order 1', async ({ page }) => {});
+  test('in order 2', async ({ page }) => {});
+});
+```
+
```

Comment:
I like mentioning it, but I think the first snippet makes this verbose. What if we replaced it with a link, like this:

```md
If your configuration applies parallel mode to all tests using [`property: TestConfig.fullyParallel`], you might still want to run some tests with default settings. You can override the mode per describe: 
[code snippet]
```

Reply:
Agreed, it's redundant!